src/python/director/startup.py

```diff
@@ -1,6 +1,8 @@
 # This script is executed in the main console namespace so
 # that all the variables defined here become console variables.
 
+from __future__ import division
```

Comment:
It turns out this did *not* extend to the console, as much as I'd liked it to as future imports are on a per module basis (cf. [here](https://docs.python.org/2/reference/simple_stmts.html#future)). You can test it by pulling the code, compiling and starting director, firing up the console and typing 4/3, the result is 1. I'm curious to understand the potential impact of this import (I'm happy to change the statements below to explicitly return floats on division if you prefer)

![screenshot from 2016-03-09 18 31 06](https://cloud.githubusercontent.com/assets/1664508/13646052/a8cd8c1e-e625-11e5-8e71-f6851b9bd751.png)